src/main/controllers/CaseDocumentController.ts

```diff
@@ -0,0 +1,33 @@
+import { Response } from 'express';
+
+import { AppRequest } from '../definitions/appRequest';
+import { getCaseApi } from '../services/CaseService';
+
+const { Logger } = require('@hmcts/nodejs-logging');
+
+const logger = Logger.getLogger('CaseDocumentController');
+
+export default class CaseDocumentController {
+  public async get(req: AppRequest, res: Response): Promise<void> {
+    try {
+      if (!req.params?.docId) {
+        logger.info('bad request parameter');
+        return res.redirect('/citizen-hub/' + req.session?.userCase?.id);
```

Comment:
Maybe lead them to the not-found page here too, you don't want to hide an error from the user, even if it isn't an exception.

Reply:
updated